src/printing.cpp

```
@@ -485,33 +534,43 @@ struct CommandPrinterVisitor {
     }
 
     void operator()(Call const& call) {
```

Clang (14.0.6)

[warning] method 'operator()' can be made const

(readability-make-member-function-const)